src/pybind/mgr/dashboard/controllers/crush_rule.py

```diff
@@ -11,10 +11,7 @@
 LIST_SCHEMA = {
     "rule_id": (int, 'Rule ID'),
     "rule_name": (str, 'Rule Name'),
-    "ruleset": (int, 'RuleSet related to the rule'),
     "type": (int, 'Type of Rule'),
-    "min_size": (int, 'Minimum size of Rule'),
-    "max_size": (int, 'Maximum size of Rule'),
     'steps': ([{str}], 'Steps included in the rule')
 }
 
```

Comment:
@avanthakkar would plz you mind adding the v2.0 to these endpoints? We don't have a decorator to set the version for all the endpoints of a resource, right?

Reply:
Ah yes, will do that ASAP!

Reply:
thank you!